lib/smartSurveyAPICall.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client')
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token':token, 'api_token_secret': tokenSecret});
```

Comment:
inconsistent spacing: `'api_token':token` should be `'api_token': token`.

Please consider using ESLint with a preset such as https://www.npmjs.com/package/eslint-config-airbnb-base to enforce consistent styling.